changelog/6819.doc.md

```diff
@@ -0,0 +1 @@
+Update "Validating Form Input" section to include details about the usage of `FormValidationAction` class.
```

Comment:
Let's include a sentence what this function can be used for ("makes it easier to validate form slots in custom actions")